pwndbg/gdblib/elf.py

```diff
@@ -244,15 +250,16 @@ def entry() -> int:
     # Try common names
     for name in ["_start", "start", "__start", "main"]:
         try:
-            return pwndbg.gdblib.symbol.address(name)
+            address = pwndbg.gdblib.symbol.address(name)
+            return address if address else 0
```

Comment:
Not sure about this but its probably fine?

Reply:
reverted!